ndcube/ndcube.py

```diff
@@ -230,6 +232,8 @@ def __init__(self, data, wcs=None, uncertainty=None, mask=None, meta=None,
                 global_coords = deepcopy(global_coords)
             self._global_coords = global_coords
 
+        functools.update_wrapper(NDCube.plot, self.plotter.plot)
```

Comment:
My concern with the `NDCube.plot` part here is that this applies globally to *all* ndcube instances, so if you had two instances with different plotters it would pick the docstring for both of the one that got instantiated last?

Reply:
Yes, I worried about that as well and I've pushed a better version.